x/deployment/keeper/keeper_test.go

```diff
@@ -210,6 +223,41 @@ func Test_OnDeploymentClosed(t *testing.T) {
 	})
 }
 
+func Test_CloseGroup(t *testing.T) {
+	ctx, keeper := setupKeeper(t)
+	groups := createActiveDeployment(t, ctx, keeper)
+	gIdx0, gIdx1 := 0, 1
+
+	t.Run("close-group-0", func(t *testing.T) {
+		assert.NoError(t, keeper.OnCloseGroup(ctx, groups[gIdx0]))
```

Comment:
This doesn't really need to be in a `t.Run()`.

```go
// do the action
assert.NoError(t, keeper.OnCloseGroup(ctx, groups[gIdx0]))

// assert affects of the action
```
I think you're overly-concerned about annotating failures - line numbers are enough.